src/applications/mhv/landing-page/utilities/index.js

```diff
@@ -0,0 +1,11 @@
+function getDestinationDomain(href) {
```

Comment:
I wonder if it would be better just you pass the hostname, so that we can also catch any other destinations.

Reply:
I'd think something along the lines of `(new URL(href, document.location.href)).hostname` would get you close. Some href strings could be relative paths, so the `document.location.href` would turn those into absolute URLs while not affecting absolute URLs passed in.

Reply:
I agree with this approach and have updated the function accordingly.